src/ert/scheduler/scheduler.py

```diff
@@ -105,6 +105,7 @@ def __init__(
         self._average_job_runtime: float = 0
         self._completed_jobs_num: int = 0
         self.completed_jobs: asyncio.Queue[int] = asyncio.Queue()
+        self._fmok_lock: Optional[asyncio.Lock] = None
```

Comment:
Does this need to be `Optional`, you can set it to `asyncio.Lock()` immediately I suppose (and allowing the `async with` syntax.

Reply:
Yes, can try it, but that would alter some tests - can see how many this would be if way too many than can revert.